arch/arm64/include/arch.h

```diff
@@ -52,6 +53,37 @@
 
 #endif /* CONFIG_ARCH_ADDRENV */
 
+/****************************************************************************
```

Comment:
I cannot see a reason in this patch why these were moved, arm64_arch is visible to modules in src/common. Why were these moved?

Reply:
arch/arm64/src/common/arm64_arch.h ，cannot be seen in the public code.
eg.
irq_csection.c ...

Reply:
Ok